### ISSUE TITLE
client crashing at login

### ISSUE THREAD

**Issue by marcostrullato** [nextcloud/desktop#1110]
Hi all,
the client running in a fresh installed and updated Ubuntu 18.10 x64 is crashing, either from the appimage or from the repository.

The client vesion is 2.5.1-20181204.111806~cosmic.
It's crashing with this output, just after entering the server url and clicking on Next.
```
[8245:8245:0224/204250.357035:ERROR:gpu_process_transport_factory.cc(1019)] Lost UI shared context.
Received signal 6
#0 0x7f0d383119ee <unknown>
#1 0x7f0d38311b00 <unknown>
#2 0x7f0d38312137 <unknown>
#3 0x7f0d3212b100 <unknown>
#4 0x7f0d3212b077 gsignal
#5 0x7f0d3210c535 abort
#6 0x7f0d326868d7 QMessageLogger::fatal()
#7 0x7f0d3125bcd2 <unknown>
#8 0x7f0d3125bffa <unknown>
#9 0x7f0d3125e186 QQuickWidget::resizeEvent()
#10 0x7f0d3ca4296d <unknown>
#11 0x7f0d332701be QWidget::event()
#12 0x7f0d3125ee2d QQuickWidget::event()
#13 0x7f0d3ca42bf0 <unknown>
#14 0x7f0d332314a1 QApplicationPrivate::notify_helper()
#15 0x7f0d33238ae0 QApplication::notify()
#16 0x7f0d32870499 QCoreApplication::notifyInternal2()
#17 0x7f0d3326829b QWidgetPrivate::sendPendingMoveAndResizeEvents()
#18 0x7f0d3326c1a3 QWidgetPrivate::show_helper()
#19 0x7f0d3326c142 QWidgetPrivate::showChildren()
#20 0x7f0d3326c1bf QWidgetPrivate::show_helper()
#21 0x7f0d3326f37b QWidget::setVisible()
#22 0x7f0d3326c158 QWidgetPrivate::showChildren()
#23 0x7f0d3326c1bf QWidgetPrivate::show_helper()
#24 0x7f0d3326f37b QWidget::setVisible()
#25 0x7f0d3326c158 QWidgetPrivate::showChildren()
#26 0x7f0d3326c1bf QWidgetPrivate::show_helper()
#27 0x7f0d3326c142 QWidgetPrivate::showChildren()
#28 0x7f0d3326c1bf QWidgetPrivate::show_helper()
#29 0x7f0d3326c142 QWidgetPrivate::showChildren()
#30 0x7f0d3326c1bf QWidgetPrivate::show_helper()
#31 0x7f0d3326f37b QWidget::setVisible()
#32 0x7f0d33455b9c <unknown>
#33 0x7f0d33459c5b <unknown>
#34 0x7f0d33459fd3 QWizard::next()
#35 0x7f0d32899830 QMetaObject::activate()
#36 0x7f0d3cc0138e OCC::DetermineAuthTypeJob::authType()
#37 0x7f0d3cb7ff74 OCC::DetermineAuthTypeJob::checkBothDone()
#38 0x7f0d3cb80089 <unknown>
#39 0x7f0d32899830 QMetaObject::activate()
#40 0x7f0d3cc013ef OCC::SimpleNetworkJob::finishedSignal()
#41 0x7f0d3cb8031d OCC::SimpleNetworkJob::finished()
#42 0x7f0d3cb7b3c4 OCC::AbstractNetworkJob::slotFinished()
#43 0x7f0d32899830 QMetaObject::activate()
#44 0x7f0d337d5ae4 <unknown>
#45 0x7f0d33877309 <unknown>
#46 0x7f0d32899f92 QObject::event()
#47 0x7f0d332314a1 QApplicationPrivate::notify_helper()
#48 0x7f0d33238ae0 QApplication::notify()
#49 0x7f0d32870499 QCoreApplication::notifyInternal2()
#50 0x7f0d3287348b QCoreApplicationPrivate::sendPostedEvents()
#51 0x7f0d328c2b23 <unknown>
#52 0x7f0d31739c3e g_main_context_dispatch
#53 0x7f0d31739ed8 <unknown>
#54 0x7f0d31739f6c g_main_context_iteration
#55 0x7f0d328c2143 QEventDispatcherGlib::processEvents()
#56 0x7f0d1cd88e51 <unknown>
#57 0x7f0d3286f16b QEventLoop::exec()
#58 0x7f0d328772e2 QCoreApplication::exec()
#59 0x555e7f98704c main
#60 0x7f0d3210e09b __libc_start_main
#61 0x555e7f98750a _start
  r8: 0000000000000000  r9: 00007ffc1a9ce4e0 r10: 0000000000000008 r11: 0000000000000246
 r12: 00007ffc1a9ce828 r13: 0000555e80bf63e0 r14: 0000000000000001 r15: 00007ffc1a9cedd0
  di: 0000000000000002  si: 00007ffc1a9ce4e0  bp: 0000555e80bf63e0  bx: 00007ffc1a9ce738
  dx: 0000000000000000  ax: 0000000000000000  cx: 00007f0d3212b077  sp: 00007ffc1a9ce4e0
  ip: 00007f0d3212b077 efl: 0000000000000246 cgf: 002b000000000033 erf: 0000000000000000
 trp: 0000000000000000 msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.
```

Regards

**Comment by marcostrullato** [comment]
Hi all,

I just noticed it might be related to the nvidia propietary driver. If I switch to the open source one, the client works as expected.

Regards

**Comment by dhomoney** [comment]
I can confirm the same is happening to me on Manjaro fresh install as of 1330 CDT. Here is the output I get upon crash. As an aside, this was working just fine on an Antergos running KDE as of a couple of days ago, but that was installed back in January. 

./Nextcloud-2.5.1-x86_64.AppImage 
[14610:14610:0317/092127.010787:ERROR:gpu_process_transport_factory.cc(1019)] Lost UI shared context.
Received signal 6
#0 0x7f65d3e196af <unknown>
#1 0x7f65d28488c5 <unknown>
#2 0x7f65d3e19bbe <unknown>
#3 0x7f65d0832e00 <unknown>
#4 0x7f65d0832d7f __GI_raise
#5 0x7f65d081d672 __GI_abort
#6 0x7f65d0da37fd QMessageLogger::fatal()
#7 0x7f65cf6ed2dc <unknown>
#8 0x7f65cf6ed50a <unknown>
#9 0x7f65cf6ef8d6 QQuickWidget::resizeEvent()
#10 0x7f65d8e03d8d <unknown>
#11 0x7f65d1a5890e QWidget::event()
#12 0x7f65cf6f06eb QQuickWidget::event()
#13 0x7f65d8e04020 <unknown>
#14 0x7f65d1a18c4c QApplicationPrivate::notify_helper()
#15 0x7f65d1a202c7 QApplication::notify()
#16 0x7f65d0f9ead8 QCoreApplication::notifyInternal2()
#17 0x7f65d1a50902 QWidgetPrivate::sendPendingMoveAndResizeEvents()
#18 0x7f65d1a546f3 QWidgetPrivate::show_helper()
#19 0x7f65d1a54677 QWidgetPrivate::showChildren()
#20 0x7f65d1a5470f QWidgetPrivate::show_helper()
#21 0x7f65d1a579fd QWidget::setVisible()
#22 0x7f65d1a54688 QWidgetPrivate::showChildren()
#23 0x7f65d1a5470f QWidgetPrivate::show_helper()
#24 0x7f65d1a579fd QWidget::setVisible()
#25 0x7f65d1a54688 QWidgetPrivate::showChildren()
#26 0x7f65d1a5470f QWidgetPrivate::show_helper()
#27 0x7f65d1a54677 QWidgetPrivate::showChildren()
#28 0x7f65d1a5470f QWidgetPrivate::show_helper()
#29 0x7f65d1a54677 QWidgetPrivate::showChildren()
#30 0x7f65d1a5470f QWidgetPrivate::show_helper()
#31 0x7f65d1a579fd QWidget::setVisible()
#32 0x7f65d1c43cfc <unknown>
#33 0x7f65d1c4867c <unknown>
#34 0x7f65d1c48947 QWizard::next()
#35 0x559220af0251 OCC::OwncloudWizard::setAuthType()
#36 0x559220a6128c _ZN9QtPrivate11FunctorCallINS_11IndexesListIJLi0EEEENS_4ListIJN3OCC20DetermineAuthTypeJob8AuthTypeEEEEvMNS4_14OwncloudWizardEFvS6_EE4callESA_PS8_PPv
#37 0x559220a60ade _ZN9QtPrivate15FunctionPointerIMN3OCC14OwncloudWizardEFvNS1_20DetermineAuthTypeJob8AuthTypeEEE4callINS_4ListIJS4_EEEvEEvS6_PS2_PPv
#38 0x559220a605d2 _ZN9QtPrivate11QSlotObjectIMN3OCC14OwncloudWizardEFvNS1_20DetermineAuthTypeJob8AuthTypeEENS_4ListIJS4_EEEvE4implEiPNS_15QSlotObjectBaseEP7QObjectPPvPb
#39 0x7f65d0fcbb56 QMetaObject::activate()
#40 0x7f65d952c439 OCC::DetermineAuthTypeJob::authType()
#41 0x7f65d948769b OCC::DetermineAuthTypeJob::checkBothDone()
#42 0x7f65d9486e3a <unknown>
#43 0x7f65d9489629 <unknown>
#44 0x7f65d94894c0 <unknown>
#45 0x7f65d9489302 <unknown>
#46 0x7f65d0fcbb56 QMetaObject::activate()
#47 0x7f65d952c760 OCC::SimpleNetworkJob::finishedSignal()
#48 0x7f65d94878cc OCC::SimpleNetworkJob::finished()
#49 0x7f65d947a576 OCC::AbstractNetworkJob::slotFinished()
#50 0x7f65d947d846 _ZN9QtPrivate11FunctorCallINS_11IndexesListIJEEENS_4ListIJEEEvMN3OCC18AbstractNetworkJobEFvvEE4callES8_PS6_PPv
#51 0x7f65d947d737 _ZN9QtPrivate15FunctionPointerIMN3OCC18AbstractNetworkJobEFvvEE4callINS_4ListIJEEEvEEvS4_PS2_PPv
#52 0x7f65d947d66e _ZN9QtPrivate11QSlotObjectIMN3OCC18AbstractNetworkJobEFvvENS_4ListIJEEEvE4implEiPNS_15QSlotObjectBaseEP7QObjectPPvPb
#53 0x7f65d0fcbb56 QMetaObject::activate()
#54 0x7f65d1fcc017 <unknown>
#55 0x7f65d2067c79 <unknown>
#56 0x7f65d0fcc869 QObject::event()
#57 0x7f65d1a18c4c QApplicationPrivate::notify_helper()
#58 0x7f65d1a202c7 QApplication::notify()
#59 0x7f65d0f9ead8 QCoreApplication::notifyInternal2()
#60 0x7f65d0fa16de QCoreApplicationPrivate::sendPostedEvents()
#61 0x7f65d0ff6263 <unknown>
  r8: 0000000000000000  r9: 00007ffc285ac440 r10: 0000000000000008 r11: 0000000000000246
 r12: 00007ffc285ac780 r13: 00005592215c46b0 r14: 00007ffc285ae350 r15: 00007ffc285acd50
  di: 0000000000000002  si: 00007ffc285ac440  bp: 00005592215c46b0  bx: 0000000000000006
  dx: 0000000000000000  ax: 0000000000000000  cx: 00007f65d0832d7f  sp: 00007ffc285ac440
  ip: 00007f65d0832d7f efl: 0000000000000246 cgf: 002b000000000033 erf: 0000000000000000
 trp: 0000000000000000 msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.
[14672:14672:0100/000000.820088:ERROR:broker_posix.cc(43)] Invalid node channel message

**Comment by dhomoney** [comment]
Forgot uname -a output:

Linux desktop-manjaro 4.19.4-1-MANJARO #1 SMP PREEMPT Fri Nov 23 18:23:26 UTC 2018 x86_64 GNU/Linux

**Comment by dhomoney** [comment]
This seems to be an issue with the 4.19.x branch of the Linux kernel. In trying to debug the issue, I changed to the 4.20.15 kernel and the problem went away. I have a feeling this is also not an issue in the 5.0.x kernel either. 

uname -a output:   Linux desktop-manjaro 4.20.15-1-MANJARO #1 SMP PREEMPT Sun Mar 10 08:50:44 UTC 2019 x86_64 GNU/Linux

**Comment by faulix** [comment]
Same happens with Windows 10, so it doesn't seem to be an OS issue.

**Comment by mcnesium** [comment]
I can approve connection fails, client freezes, on Manjaro with Kernel 4.19.32. It works as expected with Kernel 5.0.5.

**Comment by claell** [comment]
This looks like a duplicate of #1036. Can you please verify that and close if appropriate?

**Comment by mcnesium** [comment]
Just because issues are not being worked on or talked about, they don't go away. Why are they being auto-closed?

**Comment by FlexW** [comment]
@mcnesium Do you still have the issue? The issue is pretty old.

**Comment by mcnesium** [comment]
@FlexW Haven't seen it in a while, but that is not the point.

People voluntarily invest time and effort to file issues as their contribution to projects. It is pretty rude imho to have them closed by bots without a comment of what the current status is. At Nextcloud, you seem to do this a lot with issues contributed by the community, and that is really discouraging for people who want to help improving the software.

**Comment by FlexW** [comment]
@mcnesium I can understand you and the frustration. When this bug was filled, I and most of the other team members were not working for Nextcloud. We currently try to catch up with all the bugs that are partially really old, and never got attention, and prioritize them, to see where we need to spend time. The bot helps us to see which bugs are already solved or duplicates of others. We also need to have steps to reproduce the problem. Otherwise, it will be very difficult/impossible to solve them. Diagnosing the cause of a crash where you just have a small crash dump, is an art itself. It's even more difficult if the version of the software where the crash was reported is pretty old, and a big part of the code has been replaced in the meantime. We solved a lot of other crash issues in the meantime, and it could very well be that this crash was also resolved.
